strings/names/cats.txt

```diff
@@ -2,13 +2,15 @@ names@=
 	Gary@,Mittens@,Mr. Jingles@,Rex@,Jasmine@,Litterbox
 	Reginald@,Poosycat@,Dr. Purrsworthy@,Lt. Scratches@,Michael Catson
 	Fluffy@,Mr. Purrfect@,Lord Furstooth@,Lion-O@,Johnathan@,Gary Catglitter
-	Chat de Gaulle@,Ratbag
+	Chat de Gaulle@,Ratbag@,Mitts@,Mr. Fluffington@,Meowxie
 	Baron Fuzzykins, Defiler of Carpets@,Robert Meowgabe@,Chairman Meow@,Bacon
-	Prunella@,Poonella@,SEXCOPTER@,Fat, Lazy Piece of Shit@,Jones Mk. II
-	Jones Mk. III@,Jones Mk. IV@,Jones Mk.V@,Mr. Meowgi
-	Furrston von Purringsworth@,Garfadukecliff@,SyndiCat@,Rosa Fluffemberg
+	Prunella@,Poonella@,SEXCOPTER@,Fat, Lazy Piece of Shit@,Jones Mk. II@,Fish eater
+	Jones Mk. III@,Jones Mk. IV@,Jones Mk.V@,Mr. Meowgi@,Meoxie@,Zeta Jones@,Double Zeta Jones@,Nu Jones
+	Furrston von Purringsworth@,Garfadukecliff@,SyndiCat@,Rosa Fluffemberg@,Graxor The Litterbox Overlord
 	Karl Meowx@,Margaret Scratcher@,Marcel Purroust@,Franz Katka@,Das Katpital
 	Proletaricat@,Perestroikat@,Mewy P. Newton@,Fidel Catstro@,George Lucats
 	Lin Miao@,Felix Purrzhinsky@,Pol Pet@,Piggy@,Long Kitty@,Caterella@,Aristocat
 	Arthur Cat Clarke@,Prick@,Pantalaimon@,Purrsula K. Le Guin@,Douglas Pat 'Ems
 	Gato@,Zebra@,Possum@,Myst@,Too@,P@,Z@,Catopus
+	Jennyanydots@,Growltiger@,Rum Tum Tugger@,Jellicle Cat@,Mungojerrie@,Rumpleteazer@,Old Deuteronomy@,The Great Rumpus Cat
+	Mr. Mistoffelees@,Macavity@,Gus The Theatre Cat@,Asaparagus@,Bustopher Jones@,Skimbleshanks@,Cat Morgan
```

Comment:
Maybe I don't get the joke, but if not that's not how you spell Asparagus

Reply:
D'oh!